docs/docs/packaging.md

```diff
@@ -85,3 +85,61 @@ With conda, dependency constraints of the form `X.Y.*` will permit the installat
 pip is stricter and will not generally install nightlies unless the `--pre` argument is specified.
 However, pip can be instructed to install nightlies of specific packages by [using any specifier containing a pre-release or development version](https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions).
 Therefore, during wheel builds we also do this [in our CI scripts by adding a `>=0.0.0a0` specifier to all the relevant constraints](https://github.com/rapidsai/cudf/blob/branch-24.06/ci/build_wheel.sh#L34).
+
+## `dependencies.yaml` Conventions
```

Comment:
Other conventions that we ought to document:
- `depends_on_X`
- `--extra-index-url` handling

Reply:
I did this in https://github.com/rapidsai/build-planning/pull/107